### PR TITLE
Music loop+remix

### PR DESCRIPTION
**Pull request by RobertLevenbach**
Working, changes music@score of 500+

### CODE REVIEW

File: src/main/java/tetris/Controller.java [hunk 1]
```diff
@@ -73,6 +77,25 @@ public void handleKeyEvent(KeyEvent event) {
         redraw();
     }
 
+    public void changingMusic() {
+        System.out.println(score.getScore());
```
Comment (pascal08) [inline src/main/java/tetris/Controller.java]: Could you use the Logger for this?

File: src/main/java/tetris/Score.java [hunk 2]
```diff
@@ -26,6 +27,7 @@ public void reset() {
     public void add(int rowsCleared) {
         score += rowsCleared * pointsPerRow[rowsCleared];
         level += rowsCleared;
+        score4music += rowsCleared * pointsPerRow[rowsCleared];
```
Comment (pascal08) [inline src/main/java/tetris/Score.java]: Why do you need this variable if it holds exactly the same value as score?

File: src/main/java/tetris/sound/AudioStreaming.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        Media l = new Media("file:///" + System.getProperty("user.dir").replace('\\', '/') + "/"
+            + "src/main/resources/sound/theme.mp3");
+        player = new MediaPlayer(l);
+        player.setCycleCount(themeLoop);
+        player.play();
+    }
+
+    public static void playRemix() {
+        player.stop();
+        Media m = new Media("file:///" + System.getProperty("user.dir").replace('\\', '/') + "/"
```
Comment (pascal08) [inline src/main/java/tetris/sound/AudioStreaming.java]: I believe it is better to use a relative path (i.e getClass().getResource())

File: src/main/java/tetris/sound/AudioStreaming.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package tetris.sound;
+
+import javafx.scene.media.Media;
+import javafx.scene.media.MediaPlayer;
+
+public class AudioStreaming {
+    private static int themeLoop = MediaPlayer.INDEFINITE;
+    public static MediaPlayer player;
+    public static MediaPlayer player2;
```
Comment (pascal08) [inline src/main/java/tetris/sound/AudioStreaming.java]: Maybe rename this to remixPlayer? That would be more descriptive.

File: src/main/java/tetris/Controller.java (hunk 1 above)
Comment (RobertLevenbach) [inline src/main/java/tetris/Controller.java]: yes, this was just for testing when the music changed, so I could listen if it actually changed(forgot to remove it afterwards)

File: src/main/java/tetris/sound/AudioStreaming.java (hunk 3 above)
Comment (RobertLevenbach) [inline src/main/java/tetris/sound/AudioStreaming.java]: I tried that, only that method didn't work for me(also getResourceAsStream didn't work)... Isn't this method universal??

**Comment by RobertLevenbach** [comment]
Thanks for the review, will change the mistakes @pascal08

**Pull request closed without merging**